src/box2dx/Box2D.NetStandard/Box2D.NetStandard.csproj

```diff
@@ -3,15 +3,17 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Authors>Erin Catto</Authors>
-    <Copyright>Copyright © Ihar Kalasouski 2008</Copyright>
-    <PackageTags>Box2D, Physics</PackageTags>
-    <Description>Box2D is the C# port of C++ Box2D. This package supports .NET Standard 2.1.</Description>
+    <Authors>Erin Catto, Ben Ukhanov, Hugh Phoenix-Hulme</Authors>
+    <Copyright>Copyright © Erin Catto, Ben Ukhanov, Hugh Phoenix-Hulme 2020</Copyright>
+    <PackageTags>Box2D, Physics, netcore, netstandard</PackageTags>
```

Comment:
Can you please change to: <PackageTags>Box2D, Physics, .NET, Standard, Core</PackageTags>

Reply:
NVM, I changed and will push the changes to master.

Reply:
Pushed.